crates/credentials/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "warg-credentials"
+version = { workspace = true}
+authors = { workspace = true}
+edition = { workspace = true}
+rust-version = { workspace = true}
+license = { workspace = true}
+homepage = { workspace = true}
+repository = { workspace = true}
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true}
+indexmap = { workspace = true}
+keyring = "2.3.2"
```

Comment:
Nit: although this is probably the only crate that would take this dependency, I've generally favored having dependency versions kept at the workspace-level.